support_branding/models/res_company.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Sunflower IT
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import AccessError
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    @api.model
+    def get_support_branding_config_param_data(self):
+        if not self.env.user.has_group("base.group_user"):
+            raise AccessError(
+                _(
+                    "You are not allowed to access this "
+                    "functionality, please contact Admin for "
+                    "more support"
+                )
+            )
+        self.env.cr.execute(
+            "select key, value from ir_config_parameter where key ilike "
+            "'%support_%';"
```

Comment:
Also, the parameters are saved like so: `support_...`, so the `%` in the beginning _could_ be removed, but it is not a big deal. However, perhaps it would have been more neat if it was saved as `support_branding.support_...`, but that is another issue.